docs/ops/normalization/BatchNormInference_1.md

```diff
@@ -4,39 +4,33 @@
 
 **Category**: *Normalization*
 
-**Short description**: *BatchNormInference* layer normalizes a `input` tensor by `mean` and `variance`, and applies a scale (`gamma`) to it, as well as an offset (`beta`).
+**Short description**: *BatchNormInference* performs Batch Normalization operation described in the [Batch Normalization: Accelerating Deep Network Training by Reducing Internal Covariate Shift](https://arxiv.org/abs/1502.03167v2) article.
 
-**Attributes**:
+**Detailed Description**
 
-* *epsilon*
-  * **Description**: *epsilon* is the number to be added to the variance to avoid division by zero when normalizing a value. For example, *epsilon* equal to 0.001 means that 0.001 is added to the variance.
-  * **Range of values**: a positive floating-point number
-  * **Type**: `float`
-  * **Default value**: None
-  * **Required**: *yes*
+*BatchNormInference* performs the following operations on a given data batch input tensor `data`:
 
-**Inputs**
+* Normalize each activation \f$x^{(k)}\f$ by the mean and variance.
```

Comment:
Normalize each activation --> Normalizes each activation \f$x^{(k)}\f$ by the mean and variance.
Can we make this in 3d person so it's more consistent? Example: Performs operations: - Normalizes... -Performs linear transformation

Reply:
Absolutely, I am sorry I missed it in the previous comment